src/models/internal-table.ts

```diff
@@ -27,4 +27,5 @@ export type TableStyleDetails = {
   headerBottom: TableLineDetails;
   tableBottom: TableLineDetails;
   vertical: string;
+  rowBottom: TableLineDetails;
```

Comment:
rowSeperator is a better name I think.

Reply:
I wasn't convinced about that name myself, I will rename to rowSeparator